templates/supervisor/celery_network.j2

```diff
@@ -0,0 +1,11 @@
+[program:celery_network]
+user={{ www_user }}
+directory={{ openwisp2_path }}
+command={{ openwisp2_path }}/env/bin/celery -A openwisp2 worker -l info --queue network -O {{ openwisp2_celery_network_optimization }}{% if openwisp2_celery_network_concurrency %} --concurrency={{ openwisp2_celery_network_concurrency }}{% elif openwisp2_celery_network_autoscale %} --autoscale {{ openwisp2_celery_network_autoscale }}{% endif %}{% if openwisp2_celery_network_prefetch_multiplier %} --prefetch-multiplier {{ openwisp2_celery_network_prefetch_multiplier }}{% endif %} -n network@%%h
```

Comment:
would be great to know if there's a way to break this long line into multiple lines